apply-templates.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+set -Eeuo pipefail
+
+[ -f versions.json ] # run "versions.sh" first
+
+jqt='.jq-template.awk'
+if [ -n "${BASHBREW_SCRIPTS:-}" ]; then
+	jqt="$BASHBREW_SCRIPTS/jq-template.awk"
```

Comment:
In the end are we doing anything with Bashbrew? or can we remove these lines?

Reply:
As I've gone through the file more, I see this creates the `.jq-template.awk` but I'm not clear on use here just yet. Just triple checking this is all needed or not